build.gradle

```diff
@@ -21,12 +22,12 @@ repositories {
 
 dependencies {
     implementation "ru.vyarus:gradle-quality-plugin:4.7.0"
-    implementation 'com.github.spotbugs.snom:spotbugs-gradle-plugin:5.0.9'
-    implementation ("com.diffplug.spotless:spotless-plugin-gradle:6.6.0") {
+    implementation 'com.github.spotbugs.snom:spotbugs-gradle-plugin:5.0.12'
+    implementation ("com.diffplug.spotless:spotless-plugin-gradle:6.11.0") {
         exclude group: "com.google.googlejavaformat", module: "google-java-format-parent"
     }
     implementation "com.google.googlejavaformat:google-java-format-parent:1.15.0"
-    implementation "io.gitlab.arturbosch.detekt:detekt-gradle-plugin:1.20.0"
+    implementation "io.gitlab.arturbosch.detekt:detekt-gradle-plugin:1.22.0-RC1"
```

Comment:
I'm always paranoid about release candidates. Is there something we need from `1.22.0` specifically? I definitely trust your judgment, just curious.

Reply:
If it was production code, I would agree. I'm on the fence with a linter. 🤷